CharlieBackend.Core/DTO/Dashboard/StudentGroupResults/StudentsGroupsResultsDto.cs

```diff
@@ -7,5 +7,12 @@ public class StudentGroupsResultsDto
         public IEnumerable<AverageStudentGroupVisitDto> AverageStudentGroupsVisits { get; set; }
```

Comment:
please initialize list here instead of constructor: public IEnumerable AverageStudentGroupsVisits { get; set; } = new List<AverageStudentGroupVisitDto>();